src/routes/(console)/onboarding/create-project/+page.svelte

```diff
@@ -0,0 +1,170 @@
+<script lang="ts">
+    import { Card, Layout, Typography, Input, Tag, Icon, Button } from '@appwrite.io/pink-svelte';
+    import { IconPencil } from '@appwrite.io/pink-icons-svelte';
+    import { CustomId } from '$lib/components/index.js';
+    import type { RegionList } from '$lib/sdk/billing';
+    import { onMount } from 'svelte';
+    import { isCloud } from '$lib/system';
+    import { sdk } from '$lib/stores/sdk';
+    import { isValueOfStringEnum } from '$lib/helpers/types';
+    import { Flag, ID, Region } from '@appwrite.io/console';
+    import Loading from './loading.svelte';
+    import { BillingPlan } from '$lib/constants';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { goto } from '$app/navigation';
+    import { base } from '$app/paths';
+
+    let showCustomId = false;
+    let isLoading = false;
+    let id: string;
+    let startAnimation = false;
+    let projectName = '';
+
+    let regions: RegionList;
+    onMount(async () => {
+        if (isCloud) {
+            regions = await sdk.forConsole.billing.listRegions();
+            regions.regions.forEach((region) => {
+                const img = new Image();
+                img.src = getFlagUrl(region.flag);
+            });
+        }
+    });
+
+    function getFlagUrl(countryCode: string) {
+        if (!isValueOfStringEnum(Flag, countryCode)) return '';
+        return sdk.forProject.avatars.getFlag(countryCode, 22, 15, 100)?.toString();
+    }
+
+    async function createProject() {
+        isLoading = true;
+
+        const org = await sdk.forConsole.billing.createOrganization(
+            ID.unique(),
+            'Personal Projects',
+            BillingPlan.FREE,
+            null,
+            null
+        );
+
+        const teamId = org.$id;
+        try {
+            const project = await sdk.forConsole.projects.create(
+                id ?? ID.unique(),
+                projectName,
+                teamId,
+                Region.Default
+            );
+            trackEvent(Submit.ProjectCreate, {
+                customId: !!id,
+                teamId
+            });
+
+            startAnimation = true;
+
+            setTimeout(() => {
+                goto(`${base}/project-${project.$id}`);
+            }, 3000);
+        } catch (e) {
+            // error = e.message;
+            trackError(e, Submit.ProjectCreate);
+            isLoading = false;
```

Comment:
lets throw an error notification. I tried to see this locally, and it just kept on loading and no error was indicated to me. We should also probably redirect from the animation in case of an error, wdyt?